src/Symfony/Bundle/FrameworkBundle/Resources/config/routing.xml

```diff
@@ -40,7 +40,7 @@
             <argument type="service" id="file_locator" />
         </service>
 
-        <service id="routing.loader.service" class="Symfony\Component\Routing\Loader\DependencyInjection\ServiceRouterLoader">
+        <service id="routing.loader.service" class="Symfony\Component\Routing\Loader\ContainerLoader">
```

Comment:
I just thought about the fact that `ContainerLoader` does not extends `ServiceRouterLoader`. This public service needs a BC layer? How can I do it efficiently?

Reply:
ContainerLoader should extend ServiceRouterLoader then

Reply:
Or deprecate the service in favor of a `routing.loader.container` one, removing the `routing.loader` tag from this one?

Reply:
Renaming it is easier and prevents the new `ContainerLoader` to have deprecated content, so I did it.